### ISSUE TITLE
Support for Symfony 6

### ISSUE THREAD

**Issue by tarlepp** [matthiasnoback/symfony-console-form#50]
Symfony 6 was released, so I think that we should create new version of this awesome library for Symfony 6 and drop support for Symfony 5 on that version - WDYT?

**Comment by matthiasnoback** [comment]
Of course! Please go ahead when you have time for it.

**Comment by tarlepp** [comment]
Yeah, we need to wait that
* https://packagist.org/packages/friendsofphp/php-cs-fixer

has support for Symfony 6 first, there is open PR for that already;
* https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6095

**Comment by tarlepp** [comment]
Meanwhile, I looked that `docker-compose.yml` and there are those `init: true` definitions, looks like that isn't supported anymore or at least my IDE gives error from those - what is the purpose of those?

And it would be really nice if there was some documentation how to use Docker within this project - WDYT?

**Comment by matthiasnoback** [comment]
> Meanwhile, I looked that docker-compose.yml and there are those init: true definitions, looks like that isn't supported anymore or at least my IDE gives error from those - what is the purpose of those?

I added this earlier so you can actually Ctrl + C the PHP scripts that are running inside a container. Not sure if that works without `init: true`.

> And it would be really nice if there was some documentation how to use Docker within this project - WDYT?

Makes sense, can you add it too?

**Comment by matthiasnoback** [comment]
> Yeah, we need to wait that https://packagist.org/packages/friendsofphp/php-cs-fixer

It would make sense to switch to ECS for the code style fixer. I know its code lives completely separate from the project's code, so then there would be no need to wait.

**Comment by tarlepp** [comment]
> > Meanwhile, I looked that docker-compose.yml and there are those init: true definitions, looks like that isn't supported anymore or at least my IDE gives error from those - what is the purpose of those?
> 
> I added this earlier so you can actually Ctrl + C the PHP scripts that are running inside a container. Not sure if that works without `init: true`.

My IDE is giving me following;
![image](https://user-images.githubusercontent.com/595561/144850363-3d71ade2-a637-4e06-98d8-db0f75d5c8e5.png)

And when I try to run `docker-compose up` command I get;

```bash
da_wunder@wunder-VirtualBox:~/PhpstormProjects/symfony-console-form$ docker-compose up
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.composer: 'init'
Unsupported config option for services.php: 'init'
da_wunder@wunder-VirtualBox:~/PhpstormProjects/symfony-console-form$ 
```

> > And it would be really nice if there was some documentation how to use Docker within this project - WDYT?
> 
> Makes sense, can you add it too?

Sorry but I don't have clue how to use that setup within this project, so I think that is something that you need to do.

**Comment by matthiasnoback** [comment]
Thanks for describing the problems here, I'll look further into this asap.

**Comment by matthiasnoback** [comment]
Fixed by #51 When you have to time to try it, please let me know how it went. Everything should be working with both Symfony 5 & 6 on PHP 8 and higher.

**Comment by matthiasnoback** [comment]
Closing this issue now, but feel free to create new issues for potential issues you run into.
